src/schemas.py

```diff
@@ -59,79 +59,109 @@ class Distribution(BaseModel):
     media_type: str
     table_schema: TableSchema
 
-
-# If we wanted to provide the ability to attach arbitrary RDF we might want to
-# look at https://github.com/pydantic/pydantic/discussions/5853
-class Dataset(BaseModel):
-    context: Literal["https://data.ons.gov.uk/ns#"] = Field(alias="@context")
+class Edition(BaseModel):
     id: str = Field(alias="@id")
-    type: Literal["dcat:DatasetSeries"] = Field(alias="@type")
+    type: Literal["dcat:Dataset"] = Field(alias="@type")
+    in_series: str
     identifier: str
     title: str = Field(max_length=90)
     summary: str = Field(max_length=200)
     description: str = Field(max_length=250)
-    release_date: str = Field(
-        pattern=r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(Z|[+-]\d{2}:\d{2})?$"
-    )
-    next_release: str = Field(
-        pattern=r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(Z|[+-]\d{2}:\d{2})?$",
-    )
     publisher: str
     creator: str
     contact_point: ContactPoint
-    theme: Union[str, List[str]]
+    topics: Union[str, List[str]]
     frequency: Frequency
     keywords: list[str]
     licence: str
+    issued: str = Field(
```

Comment:
It would be good to add a `datetime.datetime.fromisoformat(dt_str)` validator instead of accepting 3000-99-32.

```python
def datetime_valid(dt_str):
    try:
        datetime.fromisoformat(dt_str)
    except:
        return False
    return True
```